README.md

```diff
@@ -1076,6 +1076,46 @@ _You can enable the following settings in Xcode by running [this script](resourc
 
   </details>
 
+* <a id='omit-single-expression-return'></a>(<a href='#omit-single-expression-return'>link</a>) **Omit the `return` keyword when not required by the language.** [![SwiftFormat: redundantReturn](https://img.shields.io/badge/SwiftFormat-redundantReturn-7B0051.svg)](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#redundantReturn)
```

Comment:
@calda should we update the link too, now that we've simplified the language?

Reply:
To just `#omit-return`?

Reply:
That’s consistent with the `self` one, so yeah